lldb/test/API/lang/swift/foundation_value_types/uuid/main.swift

```diff
@@ -12,6 +12,7 @@
 import Foundation
 
 func main() {
+  //% self.runCmd("settings set target.experimental.swift-read-metadata-from-file-cache false")
```

Comment:
Why is this one commented out?

Reply:
that's the syntax for those "inlined" tests

Reply:
Oh right